packages/plugin-core/src/components/lookup/NotePickerUtils.ts

```diff
@@ -111,27 +106,15 @@ export class NotePickerUtils {
       // @ts-ignore
       vault: {},
     });
-    const ABUserGroup =
-      _2022_09_CREATE_NEW_WITH_TEMPLATE_QUICKPICK_LABEL_HIGHLIGHT_TEST.getUserGroup(
-        SegmentClient.instance().anonymousId
-      );
 
-    let label: string;
-    if (
-      ABUserGroup ===
-      CreateNewWithTemplateQuickPickLabelHighlightTestGroups.label
-    ) {
-      label = LabelUtils.createLabelWithHighlight({
-        value: CREATE_NEW_WITH_TEMPLATE_LABEL,
-        highlight: {
-          value: "$(beaker) [New] ",
-          location: "prefix",
-          expirationDate: new Date("2022-11-01"),
-        },
-      });
-    } else {
-      label = CREATE_NEW_WITH_TEMPLATE_LABEL;
-    }
+    const label = LabelUtils.createLabelWithHighlight({
+      value: CREATE_NEW_WITH_TEMPLATE_LABEL,
+      highlight: {
+        value: "$(beaker) [New] ",
+        location: "prefix",
+        expirationDate: new Date("2022-11-15"),
```

Comment:
Apart from the removal of the A/B test what is the code the is the fix? I could only pattern-match the change of the date in expirationDate.

Reply:
The ab test was defined and depending on the test group `createNewWithTemplateItem` will return a highlighted label or just a regular label

because I defined the ab test but not actually export it out, nobody actually got divided in the split tests so nobody actually got the highlighted labels.

We determined that ab test was unnecessary for this feature and removed it.

Now that we removed it, everyone will get the highlighted label. (which is what I'm describing as a fix)